tests/end_to_end/test_osquery_integration/data/playbooks/configuration.yaml

```
@@ -0,0 +1,62 @@
+- name: Test configuration
+  hosts: wazuh-manager
+  become: true
+  tasks:
+
+    - name: Check if Osquery is installed or not on CentOS
+      command: systemctl status osqueryd --no-pager
+      register: osquery_check
+      when: ansible_facts['distribution'] == "CentOS"
+      ignore_errors: true
```

Put a comment indicating why it is necessary to ignore the errors in this task